yarn-project/archiver/src/archiver/archiver_store.ts

```diff
@@ -15,23 +15,148 @@ import { L1ToL2MessageStore, PendingL1ToL2MessageStore } from './l1_to_l2_messag
+   */
   getL2Blocks(from: number, take: number): Promise<L2Block[]>;
-  addUnverifiedData(data: UnverifiedData[]): Promise<boolean>;
+
+  /**
+   * Append new Unverified data to the store's list.
+   * @param data - The Unverified Data to be added to the store.
+   * @returns True if the operation is successful.
+   */
+  addUnverifiedData(data: EventLogs[]): Promise<boolean>;
+
+  /**
+   * Append new encrypted event logs data to the store's list.
+   * @param data - The Unverified Data to be added to the store.
+   * @returns True if the operation is successful.
+   */
+  addEncryptedEventLogs(data: EventLogs[]): Promise<boolean>;
+
+  /**
+   * Append new pending L1 to L2 messages to the store.
+   * @param messages - The L1 to L2 messages to be added to the store.
+   * @returns True if the operation is successful.
+   */
   addPendingL1ToL2Messages(messages: L1ToL2Message[]): Promise<boolean>;
+
+  /**
+   * Remove pending L1 to L2 messages from the store (if they were cancelled).
+   * @param messageKeys - The message keys to be removed from the store.
+   * @returns True if the operation is successful.
+   */
   cancelPendingL1ToL2Messages(messageKeys: Fr[]): Promise<boolean>;
+
+  /**
+   * Messages that have been published in an L2 block are confirmed.
+   * Add them to the confirmed store, also remove them from the pending store.
+   * @param messageKeys - The message keys to be removed from the store.
+   * @returns True if the operation is successful.
+   */
   confirmL1ToL2Messages(messageKeys: Fr[]): Promise<boolean>;
+
+  /**
+   * Gets the `take` amount of pending L1 to L2 messages, sorted by fee
+   * @param take - The number of messages to return (by default NUMBER_OF_L1_L2_MESSAGES_PER_ROLLUP).
+   * @returns The requested L1 to L2 message keys.
+   */
   getPendingL1ToL2MessageKeys(take: number): Promise<Fr[]>;
+
+  /**
+   * Gets the confirmed L1 to L2 message corresponding to the given message key.
+   * @param messageKey - The message key to look up.
+   * @returns The requested L1 to L2 message or throws if not found.
+   */
   getConfirmedL1ToL2Message(messageKey: Fr): Promise<L1ToL2Message>;
-  getUnverifiedData(from: number, take: number): Promise<UnverifiedData[]>;
+
+  /**
+   * Gets the `take` amount of unverified data starting from `from`.
+   * @param from - Number of the L2 block to which corresponds the first `unverifiedData` to be returned.
+   * @param take - The number of `unverifiedData` to return.
+   * @returns The requested `unverifiedData`.
+   */
+  getUnverifiedData(from: number, take: number): Promise<EventLogs[]>;
+
+  /**
+   * Gets the `take` amount of unverified data starting from `from`.
+   * @param from - Number of the L2 block to which corresponds the first `unverifiedData` to be returned.
+   * @param take - The number of `unverifiedData` to return.
+   * @returns The requested `unverifiedData`.
+   */
+  getEncryptedEventLogs(from: number, take: number): Promise<EventLogs[]>;
```

Comment:
Using `getEncryptedLogs` instead of `getEncryptedEventLogs` would be better because everywhere in the specs it's called encrypted logs. Also logs and events are used interchangeable and mean kinda the same thing in the Solidity world so having the word "event" there is redundant:
![image](https://github.com/AztecProtocol/aztec-packages/assets/13470840/5b6c9a89-d2db-46f4-a847-415b0bbfb88b)